stdlib/REPL/src/LineEdit.jl

```diff
@@ -2829,7 +2830,11 @@ function prompt!(term::TextTerminal, prompt::ModalInterface, s::MIState = init_s
         old_state = mode(s)
         while true
             kmap = keymap(s, prompt)
-            fcn = match_input(kmap, s)
+            waitany((
```

Comment:
`waitany()` isn't in 1.11 unfortunately, so this can't be easily backported unless #53341 is backported too.

Reply:
@Keno @KristofferC I think it's reasonable to backport `waitany` so I've added the label.